integration-libs/opf/components/opf-checkout-payment-wrapper/opf-checkout-payment-wrapper.service.ts

```diff
@@ -107,4 +104,36 @@ export class OpfCheckoutPaymentWrapperService {
         });
     }
   }
+
+  protected onInitPaymentError(): Observable<boolean> {
```

Comment:
In my opinion 'on' prefix might suggest that we are dealing with some event, however we are naming a method which reacts on such event. What do you tihnk about method name change to `handlePaymentInitiationError`?

Reply:
Make sense, changed